clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -74,7 +74,7 @@
 
   static {
     STAT_STORE_METHOD_REFERENCES.put("allocatedMemory", ServerStoreImpl::getAllocatedMemory);
-    STAT_STORE_METHOD_REFERENCES.put("dataOccupiedMemory", ServerStoreImpl::getDataOccupiedMemory);
+    STAT_STORE_METHOD_REFERENCES.put("dataAllocatedMemory", ServerStoreImpl::getDataAllocatedMemory);
```

Comment:
shouldn't there be a test to track the presence / absence of stats observers ?
could be done in a separate PR though